ckanext/cioos_theme/templates/base.html

```diff
@@ -22,7 +22,7 @@
     {%- block title -%}
     {%- block subtitle %}{% endblock -%}
     {%- if self.subtitle()|trim %} {{ g.template_title_deliminater }} {% endif -%}
```

Comment:
Just noticed this now, but is line 24, g.template_title_deliminater meant to be g.template_title_delimiter? I'm not familiar with the first word, or know enough about the context.

Reply:
this is from the core template and I think is correct. There is a default entry for it in the app g globals.

https://github.com/cioos-siooc/ckan/blob/eb84852e1e3791a3d802f63c15d57d4cb3c193d6/ckan/lib/app_globals.py#L45